modules/core/src/main/scala/com/azavea/stac4s/extensions/ItemExtension.scala

```diff
@@ -0,0 +1,30 @@
+package com.azavea.stac4s.extensions
+
+import com.azavea.stac4s.StacItem
+
+import io.circe._
+import io.circe.syntax._
+
+// typeclass trait for anything that is an extension of item properties
```

Comment:
This was the first one I wrote, so it's where the comments are, even though it's not first in the file order. It's pretty basic, since `StacItem`s don't have `extensionFields`, they just toss stuff into `properties`, but it explains the general strategy.